src/Controls/tests/UITests/Tests/CarouselViewUITests.cs

```diff
@@ -23,9 +23,9 @@ protected override void FixtureSetup()
 		[Category(UITestCategories.CarouselView)]
 		public async Task CarouselViewSetPosition()
```

Comment:
This one is failing on iOS
![image](https://github.com/dotnet/maui/assets/6755973/ef99c4fe-4134-47ae-ab61-1f54cc6ee0e8)